### ISSUE TITLE
Can't run MMLU with Llama2-7b quantization when GPTQ group_size is 32.

### ISSUE THREAD

**Issue by matichon-vultureprime** [NVIDIA/TensorRT-LLM#988]
### System Info

**System**
CPU architecture X86_64 (EC2 G5.8xlarge)
CPU/Host memory size 128GB, 8GB Swap
GPU properties
GPU name A10G
GPU memory size 24GB

**Libraries**
TensorRT-LLM main
TensorRT-LLM [https://github.com/NVIDIA/TensorRT-LLM/commit/b57221b764bc579cbb2490154916a871f620e2c4](https://github.com/NVIDIA/TensorRT-LLM/commit/b57221b764bc579cbb2490154916a871f620e2c4)

**Container used**
I use this script 
`git submodule update --init --recursive && git lfs install && git lfs pull && make -C docker release_build CUDA_ARCHS="86-real" && make -C docker release_run`
NVIDIA driver version 535.104.12
OS Ubuntu 20.04 (IAM ami-095ff65813edaa529)

### Who can help?

@Tracin 

### Information

- [X] The official example scripts
- [ ] My own modified scripts

### Tasks

- [X] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

1.  `python llama.py /root/.cache/huggingface/hub/models--SeaLLMs--SeaLLM-7B-Chat/snapshots/09e8d13a07d3b8b447418a3c8d18ee87591631fa/ wikitext2 --wbits 4 --true-sequential --groupsize 32 --save_safetensors ../llama-7b-4bit-gs32.safetensors `

2. `python build.py --model_dir /root/.cache/huggingface/hub/models--SeaLLMs--SeaLLM-7B-Chat/snapshots/09e8d13a07d3b8b447418a3c8d18ee87591631fa/ \
                --quant_ckpt_path ./llama-7b-4bit-gs32.safetensors \
                --dtype float16 \
                --remove_input_padding \
                --use_gpt_attention_plugin float16 \
                --enable_context_fmha \
                --use_gemm_plugin float16 \
                --use_weight_only \
                --weight_only_precision int4_gptq \
                --use_inflight_batching \
                --paged_kv_cache \
                --per_group \
                --group_size 32 \
                --vocab_size 48512 \
                --max_batch_size 8 \
                --max_input_len 4096 \
                --max_output_len 4096 \ 
                --output_dir ./seaLLM_awq_gs_32/ 
`

3.  `python mmlu.py --hf_model_dir /root/.cache/huggingface/hub/models--SeaLLMs--SeaLLM-7B-Chat/snapshots/09e8d13a07d3b8b447418a3c8d18ee87591631fa/ --engine_dir /app/tensorrt_llm/examples/llama/seaLLM_awq_gs_32/ --test_trt_llm  --data_dir ./data/mmlu`

### Expected behavior

I should run MMLU normally.

### actual behavior

```
fatal: not a git repository (or any of the parent directories): .git
[TensorRT-LLM] TensorRT-LLM version: 0.8.0.dev20240123
  0%|                                                                                                                                                                                                                                                    | 0/57 [00:00<?, ?it/s]/usr/local/lib/python3.10/dist-packages/torch/nested/__init__.py:165: UserWarning: The PyTorch API of nested tensors is in prototype stage and will change in the near future. (Triggered internally at /opt/pytorch/pytorch/aten/src/ATen/NestedTensorImpl.cpp:178.)
  return _nested.nested_tensor(
terminate called after throwing an instance of 'tensorrt_llm::common::TllmException'
  what():  [TensorRT-LLM][ERROR] Assertion failed: No valid weight only groupwise GEMM tactic(It is usually caused by the failure to execute all candidate configurations of the CUTLASS kernel, please pay attention to the warning information when building the engine.) (/src/tensorrt_llm/cpp/tensorrt_llm/plugins/weightOnlyGroupwiseQuantMatmulPlugin/weightOnlyGroupwiseQuantMatmulPlugin.cpp:386)
1       0x7f8cd2ae7233 /usr/local/lib/python3.10/dist-packages/tensorrt_llm/libs/libnvinfer_plugin_tensorrt_llm.so(+0x4b233) [0x7f8cd2ae7233]
2       0x7f8cd2bccb06 tensorrt_llm::plugins::WeightOnlyGroupwiseQuantMatmulPlugin::enqueue(nvinfer1::PluginTensorDesc const*, nvinfer1::PluginTensorDesc const*, void const* const*, void* const*, void*, CUstream_st*) + 1046
3       0x7f8cf72b6ba9 /usr/local/tensorrt/lib/libnvinfer.so.9(+0x10cdba9) [0x7f8cf72b6ba9]
4       0x7f8cf728c6af /usr/local/tensorrt/lib/libnvinfer.so.9(+0x10a36af) [0x7f8cf728c6af]
5       0x7f8cf728e320 /usr/local/tensorrt/lib/libnvinfer.so.9(+0x10a5320) [0x7f8cf728e320]
6       0x7f8d05ca05e0 /usr/local/lib/python3.10/dist-packages/tensorrt/tensorrt.so(+0xa05e0) [0x7f8d05ca05e0]
7       0x7f8d05c43433 /usr/local/lib/python3.10/dist-packages/tensorrt/tensorrt.so(+0x43433) [0x7f8d05c43433]
8       0x55635016510e python(+0x15a10e) [0x55635016510e]
9       0x55635015ba7b _PyObject_MakeTpCall + 603
10      0x556350173acb python(+0x168acb) [0x556350173acb]
11      0x556350153cfa _PyEval_EvalFrameDefault + 24906
12      0x5563501659fc _PyFunction_Vectorcall + 124
13      0x55635014e45c _PyEval_EvalFrameDefault + 2220
14      0x55635017393e python(+0x16893e) [0x55635017393e]
15      0x5563501505d7 _PyEval_EvalFrameDefault + 10791
16      0x55635017393e python(+0x16893e) [0x55635017393e]
17      0x5563501505d7 _PyEval_EvalFrameDefault + 10791
18      0x5563501659fc _PyFunction_Vectorcall + 124
19      0x556350174492 PyObject_Call + 290
20      0x5563501505d7 _PyEval_EvalFrameDefault + 10791
21      0x5563501737f1 python(+0x1687f1) [0x5563501737f1]
22      0x556350174492 PyObject_Call + 290
23      0x5563501505d7 _PyEval_EvalFrameDefault + 10791
24      0x5563501737f1 python(+0x1687f1) [0x5563501737f1]
25      0x55635014f53c _PyEval_EvalFrameDefault + 6540
26      0x55635015ac14 _PyObject_FastCallDictTstate + 196
27      0x55635017086c _PyObject_Call_Prepend + 92
28      0x55635028b700 python(+0x280700) [0x55635028b700]
29      0x55635015ba7b _PyObject_MakeTpCall + 603
30      0x556350154096 _PyEval_EvalFrameDefault + 25830
31      0x5563501659fc _PyFunction_Vectorcall + 124
32      0x55635014e26d _PyEval_EvalFrameDefault + 1725
33      0x5563501659fc _PyFunction_Vectorcall + 124
34      0x55635014e26d _PyEval_EvalFrameDefault + 1725
35      0x55635014a9c6 python(+0x13f9c6) [0x55635014a9c6]
36      0x556350240256 PyEval_EvalCode + 134
37      0x55635026b108 python(+0x260108) [0x55635026b108]
38      0x5563502649cb python(+0x2599cb) [0x5563502649cb]
39      0x55635026ae55 python(+0x25fe55) [0x55635026ae55]
40      0x55635026a338 _PyRun_SimpleFileObject + 424
41      0x556350269f83 _PyRun_AnyFileObject + 67
42      0x55635025ca5e Py_RunMain + 702
43      0x55635023302d Py_BytesMain + 45
44      0x7f8e9bc94d90 /usr/lib/x86_64-linux-gnu/libc.so.6(+0x29d90) [0x7f8e9bc94d90]
45      0x7f8e9bc94e40 __libc_start_main + 128
46      0x556350232f25 _start + 37
[ip-172-31-74-126-release:02669] *** Process received signal ***
[ip-172-31-74-126-release:02669] Signal: Aborted (6)
[ip-172-31-74-126-release:02669] Signal code:  (-6)
[ip-172-31-74-126-release:02669] [ 0] /usr/lib/x86_64-linux-gnu/libc.so.6(+0x42520)[0x7f8e9bcad520]
[ip-172-31-74-126-release:02669] [ 1] /usr/lib/x86_64-linux-gnu/libc.so.6(pthread_kill+0x12c)[0x7f8e9bd019fc]
[ip-172-31-74-126-release:02669] [ 2] /usr/lib/x86_64-linux-gnu/libc.so.6(raise+0x16)[0x7f8e9bcad476]
[ip-172-31-74-126-release:02669] [ 3] /usr/lib/x86_64-linux-gnu/libc.so.6(abort+0xd3)[0x7f8e9bc937f3]
[ip-172-31-74-126-release:02669] [ 4] /usr/lib/x86_64-linux-gnu/libstdc++.so.6(+0xa2b9e)[0x7f8dda703b9e]
[ip-172-31-74-126-release:02669] [ 5] /usr/lib/x86_64-linux-gnu/libstdc++.so.6(+0xae20c)[0x7f8dda70f20c]
[ip-172-31-74-126-release:02669] [ 6] /usr/lib/x86_64-linux-gnu/libstdc++.so.6(+0xad1e9)[0x7f8dda70e1e9]
[ip-172-31-74-126-release:02669] [ 7] /usr/lib/x86_64-linux-gnu/libstdc++.so.6(__gxx_personality_v0+0x99)[0x7f8dda70e959]
[ip-172-31-74-126-release:02669] [ 8] /usr/lib/x86_64-linux-gnu/libgcc_s.so.1(+0x16884)[0x7f8e992e8884]
[ip-172-31-74-126-release:02669] [ 9] /usr/lib/x86_64-linux-gnu/libgcc_s.so.1(_Unwind_RaiseException+0x311)[0x7f8e992e8f41]
[ip-172-31-74-126-release:02669] [10] /usr/lib/x86_64-linux-gnu/libstdc++.so.6(__cxa_throw+0x3b)[0x7f8dda70f4cb]
[ip-172-31-74-126-release:02669] [11] /usr/local/lib/python3.10/dist-packages/tensorrt_llm/libs/libnvinfer_plugin_tensorrt_llm.so(+0x4b261)[0x7f8cd2ae7261]
[ip-172-31-74-126-release:02669] [12] /usr/local/lib/python3.10/dist-packages/tensorrt_llm/libs/libnvinfer_plugin_tensorrt_llm.so(_ZN12tensorrt_llm7plugins36WeightOnlyGroupwiseQuantMatmulPlugin7enqueueEPKN8nvinfer116PluginTensorDescES5_PKPKvPKPvSA_P11CUstream_st+0x416)[0x7f8cd2bccb06]
[ip-172-31-74-126-release:02669] [13] /usr/local/tensorrt/lib/libnvinfer.so.9(+0x10cdba9)[0x7f8cf72b6ba9]
[ip-172-31-74-126-release:02669] [14] /usr/local/tensorrt/lib/libnvinfer.so.9(+0x10a36af)[0x7f8cf728c6af]
[ip-172-31-74-126-release:02669] [15] /usr/local/tensorrt/lib/libnvinfer.so.9(+0x10a5320)[0x7f8cf728e320]
[ip-172-31-74-126-release:02669] [16] /usr/local/lib/python3.10/dist-packages/tensorrt/tensorrt.so(+0xa05e0)[0x7f8d05ca05e0]
[ip-172-31-74-126-release:02669] [17] /usr/local/lib/python3.10/dist-packages/tensorrt/tensorrt.so(+0x43433)[0x7f8d05c43433]
[ip-172-31-74-126-release:02669] [18] python(+0x15a10e)[0x55635016510e]
[ip-172-31-74-126-release:02669] [19] python(_PyObject_MakeTpCall+0x25b)[0x55635015ba7b]
[ip-172-31-74-126-release:02669] [20] python(+0x168acb)[0x556350173acb]
[ip-172-31-74-126-release:02669] [21] python(_PyEval_EvalFrameDefault+0x614a)[0x556350153cfa]
[ip-172-31-74-126-release:02669] [22] python(_PyFunction_Vectorcall+0x7c)[0x5563501659fc]
[ip-172-31-74-126-release:02669] [23] python(_PyEval_EvalFrameDefault+0x8ac)[0x55635014e45c]
[ip-172-31-74-126-release:02669] [24] python(+0x16893e)[0x55635017393e]
[ip-172-31-74-126-release:02669] [25] python(_PyEval_EvalFrameDefault+0x2a27)[0x5563501505d7]
[ip-172-31-74-126-release:02669] [26] python(+0x16893e)[0x55635017393e]
[ip-172-31-74-126-release:02669] [27] python(_PyEval_EvalFrameDefault+0x2a27)[0x5563501505d7]
[ip-172-31-74-126-release:02669] [28] python(_PyFunction_Vectorcall+0x7c)[0x5563501659fc]
[ip-172-31-74-126-release:02669] [29] python(PyObject_Call+0x122)[0x556350174492]
[ip-172-31-74-126-release:02669] *** End of error message ***
Aborted (core dumped)
```

### additional notes

This error also occur with AWQ when group_size != 128

**Comment by StudyingShao** [comment]
For now, GPTQ/AWQ only supports groupsize = 64 or 128.

**Comment by StudyingShao** [comment]
Hi @matichon-vultureprime , LLaMa awq with group_size = 64 was fixed by https://github.com/NVIDIA/TensorRT-LLM/pull/1019. 
Note that the build workflow was changed. Please refer to [the README](https://github.com/NVIDIA/TensorRT-LLM/tree/main/examples/llama#awq).
Please let me know if you have other questions, or close the issue if it meets your requirements.

**Comment by kmati-dev** [comment]
> Hi @matichon-vultureprime , LLaMa awq with group_size = 64 was fixed by #1019. Note that the build workflow was changed. Please refer to [the README](https://github.com/NVIDIA/TensorRT-LLM/tree/main/examples/llama#awq). Please let me know if you have other questions, or close the issue if it meets your requirements.

Got it sir, Let's me check.
